atlascope/core/rest/endpoints/dataset_endpoints.py

```diff
@@ -24,3 +28,18 @@ def get_queryset(self):
         public_datasets = Dataset.objects.filter(public=True)
         datasets = visible_datasets | public_datasets
         return datasets.all().order_by('name')
+
+    @swagger_auto_schema(
+        request_body=no_body,
+        responses={204: 'Import successful.'},
+    )
+    @object_permission_required(model=Dataset)
+    @action(
+        detail=True,
+        methods=['POST'],
+        url_path='import',
```

Comment:
Sure, it just changes the string of the rest endpoint URL. Without this argument, the `action` deocrator would automatically use the name of the function for the URL, and our endpoint would be `POST /api/vi/datasets/{id}/perform-import`. Since I can't make the function name itself `import` (as that is a reserved python word), I use this argument to make the endpoint simpler: `POST /api/vi/datasets/{id}/import`